docs/vscode.md

```diff
@@ -69,7 +71,14 @@ Once the deployment completes, the result will be displayed in the Home view.
 
 ---
 
-### Other Views and Features
+## Other Views and Features
+
+### Deploy via Entrypoint
+
+You can deploy or create a new deployment directly from an open file. If it is
+a valid entrypoint a Posit Publisher button will appear in the editor menu.
+
+![](https://cdn.posit.co/publisher/assets/img/entrypoint-button.png)
```

Comment:
I'm unsure if I did something wrong or it just hasn't propagated to the CDN yet, but I'm getting XML errors for this image.

Reply:
Got it fixed